web/src/beta/components/fields/URLField/index.tsx

```diff
@@ -0,0 +1,153 @@
+
+  const {
+    assets,
+    isLoading,
+    hasMoreAssets,
+    searchTerm,
+    selectedAssets,
+    selectAsset,
+    handleGetMoreAssets,
+    createAssets,
+    handleSortChange,
+    handleSearchTerm,
+  } = useHooks({ workspaceId: currentWorkspace?.id });
+
+  const {
+    localSearchTerm,
+    wrapperRef,
+    onScrollToBottom,
+    sortOptions,
+    handleSearchInputChange,
+    handleUploadToAsset,
+    handleSearch,
+  } = useManageAssets({
+    selectedAssets,
+    searchTerm,
+    isLoading,
+    hasMoreAssets,
+    onGetMore: handleGetMoreAssets,
+    onSortChange: handleSortChange,
+    onCreateAssets: createAssets,
+    onSearch: handleSearchTerm,
+  });
+
+  const handleClose = useCallback(() => {
+    setOpen(false);
+  }, []);
+
+  const handleClick = useCallback(() => setOpen(!open), [open]);
+
+  const handleSelect = useCallback(
+    (asset?: Asset) => {
+      if (!asset) return;
+      if (!selectedAssets.includes(asset)) selectAsset([asset]);
+      else selectAsset([]);
+    },
+    [selectedAssets, selectAsset],
+  );
+  const handleChange = useCallback(
+    (value?: string) => {
+      if (!value) return;
+      onChange?.(value);
+    },
+    [onChange],
+  );
+
+  return (
+    <Property name={name} description={description}>
+      <StyledTextField
+        value={value !== null ? t("Field set") : undefined}
+        onChange={onChange}
+        placeholder={t("Not set")}
+      />
+      {fileType === "Asset" && (
+        <ButtonWrapper>
+          <AssetButton
+            icon={assetType === "Image" ? "imageStoryBlock" : "file"}
+            text={t("Choose")}
+            iconPosition="left"
+            onClick={handleClick}
+          />
+          <AssetButton
+            icon="uploadSimple"
+            text={t("Upload")}
+            iconPosition="left"
+            onClick={handleUploadToAsset}
+          />
+        </ButtonWrapper>
+      )}
+      {open && (
+        <ChooseAssetModal
+          open={open}
+          onClose={handleClose}
+          assetType={assetType}
+          sortOptions={sortOptions}
+          handleSearch={handleSearch}
+          handleSearchInputChange={handleSearchInputChange}
+          localSearchTerm={localSearchTerm}
+          onGetMore={handleGetMoreAssets}
+          onScrollToBottom={onScrollToBottom}
+          selectedAssets={selectedAssets}
+          wrapperRef={wrapperRef}
+          assets={assets}
+          isLoading={isLoading}
+          hasMoreAssets={hasMoreAssets}
+          searchTerm={searchTerm}
+          onSelectAsset={handleSelect}
+          onSelect={handleChange}
+          onSortChange={handleSortChange}
+        />
+      )}
+    </Property>
+  );
+};
+
+const AssetButton = styled(Button)<{ active?: boolean }>`
+  cursor: pointer;
+  margin-left: 6px;
+  padding: 4px;
+  border-radius: 6px;
+  width: 127px;
+  color: ${props => props.theme.classic.main.text};
+
+  &:hover {
+    background: ${props => props.theme.classic.main.bg};
```

Comment:
Don't use classic theme in beta

Reply:
they should be deleted , those styles handled in button field, sorry my bad